airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationSummaryCheckActivityImpl.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.persistence.job.JobPersistence;
+import io.airbyte.persistence.job.models.AttemptNormalizationStatus;
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Singleton
+@AllArgsConstructor
+@NoArgsConstructor
+@Slf4j
+public class NormalizationSummaryCheckActivityImpl implements NormalizationSummaryCheckActivity {
+
+  @Inject
+  private JobPersistence jobPersistence;
+
+  @Override
+  @SuppressWarnings("PMD.AvoidLiteralsInIfCondition")
+  public Boolean shouldRunNormalization(final Long jobId, final Long attemptNumber, final Optional<Long> numCommittedRecords) throws IOException {
```

Comment:
Should this return `boolean` instead?  It appears we only return true/false and never null.